docs/guide/result.md

```diff
@@ -1,6 +1,6 @@
 # Result
 
-The validation result is an object containing errors' data occurred during validation.
+The validation result is an object containing errors' data that occurred during validation.
```

Comment:
```suggestion
The validation result is an object containing data of errors occurred during validation.
```

Reply:
`that` is optional, but is better to change the order like in the suggestion to avoid confusion with "data occurred during validation".